pkg/virt-handler/node-labeller/model.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ */
+
+package nodelabeller
+
+import (
+	"sync"
+
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/client-go/log"
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	device_manager "kubevirt.io/kubevirt/pkg/virt-handler/device-manager"
+)
+
+//NodeLabeller struct holds informations needed to run node-labeller
+type NodeLabeller struct {
+	kvmController    *device_manager.DeviceController
+	kubevirtInformer cache.SharedIndexInformer
+	nodeInformer     cache.SharedIndexInformer
+	clientset        kubecli.KubevirtClient
+	Queue            workqueue.RateLimitingInterface
+	host             string
+	namespace        string
+	logger           *log.FilteredLogger
+	clusterConfig    *virtconfig.ClusterConfig
+	hypervFeatures   supportedFeatures
+	capScanner       *CapScan
+	hostCapabilities supportedFeatures
+	cpuInfo          cpuInfo
+	virshCommandFN   func() (HostDomCapabilities, error)
+}
+
+type supportedFeatures struct {
+	sync.RWMutex
+	items map[string]bool
+}
+
+//hostCapabilities holds informations which provides libvirt,
+//so we don't have to call libvirt at every request
+type cpuInfo struct {
+	sync.RWMutex
+	models map[string]map[string]bool
```

Comment:
Please, create a new type or type alias for `map[string]bool` to make this more clear. For example: 
```
type cpuFeatures map[string]bool
```
Then you can reuse the type in all outer places with cpu features.